zulipterminal/cli/run.py

```diff
@@ -213,7 +213,7 @@ def parse_zuliprc(zuliprc_str: str) -> Dict[str, Any]:
         'theme': ('zt_dark', NO_CONFIG),
         'autohide': ('no_autohide', NO_CONFIG),
         'notify': ('disabled', NO_CONFIG),
-        'footlinks': ('enabled', NO_CONFIG),
```

Comment:
Do we want to leave this option in?

Reply:
We can, a bit of change in the logic. Would update it soon.